lib/opencensus/stats/recorder.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+
+require "opencensus/stats/view"
+require "opencensus/stats/view_data"
+
+module OpenCensus
+  module Stats
+    # Stats recorder.
+    #
+    # Recorder record measurement against measure for registered views
+    # Span represents a span in a trace record. Spans are contained in
+    # a trace and arranged in a forest. That is, each span may be a root span
+    # or have a parent span, and may have zero or more children.
```

Comment:
I think you forgot to delete some lines here about spans.

Reply:
Fixed